packages/arb-token-bridge-ui/src/hooks/useIsConnectedToArbitrum.ts

```diff
@@ -0,0 +1,13 @@
+import { useNetwork } from 'wagmi'
+
+import { isNetwork } from '../util/networks'
+
+export function useIsConnectedToArbitrum() {
+  const { chain } = useNetwork()
+
+  if (typeof chain === 'undefined') {
+    return undefined
+  }
+
+  return isNetwork(chain.id).isArbitrum
```

Comment:
looking at all the use cases and this hook's name
why not use `return isNetwork(chain?.id ?? -1).isArbitrum` instead?
Since "isConnectedToArbitrum" should be a boolean
and all its use cases when it's undefined is the same as when it's false
it'll save some === undefined checks

Reply:
When your wallet is not connected to any network, that's a different state than not being connected to Arbitrum, so I'd keep it as is